session-2/TaskManagementSystem/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	var (
+		projectStatus = "In Progress"
+		tasksCreated  = 25
+		isCompleted   = false
+	)
+
+	const (
+		low = iota
+		medium
+		high
+		tasksAvailable int = 100
+	)
+
+	fmt.Println(`  Welcome to the Task Management System!
+  Project start date is: 2024-09-18 00:00:00
+  Project: Task Management System`)
```

Comment:
I suggest to separate every sentence calling fmt.Println to increase  readability of the code
```suggestion
	fmt.Println("Welcome to the Task Management System!")
	fmt.Println("Project start date is: 2024-09-18 00:00:00")
	fmt.Println("Project: Task Management System")
```